content-src/main.scss

```diff
@@ -53,12 +53,23 @@ a {
   z-index: 100;
 }
 
+@keyframes fadeIn {
```

Comment:
Not 💯 % sure about this but if loading takes too long I don't see the fade in, just a sudden appearance of everything. Maybe fade happens when `render()` rendered ... nothing?